src/rendering/RenderUtil.cc

```diff
@@ -2229,6 +2312,113 @@ void RenderUtilPrivate::LowlightNode(const rendering::NodePtr &_node)
   }
 }
 
+/////////////////////////////////////////////////
+void RenderUtil::ViewTransparent(const Entity &_entity)
```

Comment:
the 3 `View***(const Entity &_entity)` functions are very similar. Is it possible to merge them or refactor some code in to functions so they can be reused?

Reply:
[`8cf713c`](https://github.com/ignitionrobotics/ign-gazebo/pull/878/commits/8cf713c668e3fd05c8742e926c1f29dac865c6fc)

We may need to make similar changes to https://github.com/ignitionrobotics/ign-gazebo/pull/861, after it is merged.